Arista/post_install.sh

```diff
@@ -1,4 +1,58 @@
-#!/usr/bin/bash
+#!/bin/bash
 
-systemctl enable telegraf
-systemctl restart telegraf
+SCRIPT_DIR=/usr/lib/telegraf/scripts
+
+function install_init {
+    cp -f $SCRIPT_DIR/init.sh /etc/init.d/telegraf
```

Comment:
Is there a reason this needs to be done in post-install script ? This can be done in the package script itself.